templates/definition/meter/vzlogger.yaml

```diff
@@ -29,3 +74,33 @@ render: |
     {{- if .scale }}
     scale: {{ .scale }}
     {{- end }}
+  currents:
```

Comment:
Gibts die Immer? Sonst bräuchte es in if

Reply:
wie macht man da ein "and" in der Bedingung? Oder sollte man das "or" verknüpfen.

Reply:
https://pkg.go.dev/text/template

Reply:
so jetzt auch mit "and"